helm/examples/local/hpcc-localfile/values.yaml

```diff
@@ -1,18 +1,26 @@
 # Default values for hpcc-localfile.
 
+# NB: hostpath and 'subPath's must exist on host
 common:
-  #The host path provides the location on the host machine that will contain the data.  The value can be overriden for
-  #each individual component
-  hostpath: "/var/lib/HPCCSystem/hpcc-data"
+  #The host path provides the location on the host machine that will contain the data.
+  hostpath: "/run/desktop/mnt/host/c/hpccdata"
+  mountPrefix: "/var/lib/HPCCSystems"
 
-dllStorage:
-  size: 3Gi
-  # hostpath: "/var/lib/HPCCSystem/hpcc-data/dlls/"
-
-dataStorage:
+planes:
+- name: dali
+  subPath: dalistorage
   size: 1Gi
-  #hostpath: "/var/lib/HPCCSystem/hpcc-data/data/"
-
-daliStorage:
+- name: dll
+  subPath: queries # cannot currently be changed
   size: 1Gi
-  #hostpath: "/var/lib/HPCCSystem/hpcc-data/dali/"
+  rwmany: true
+- name: sasha
+  subPath: sasha
+  size: 1Gi
+  rwmany: true
+- name: data
+  subPath: hpcc-data # cannot currently be changed
```

Comment:
is this true?

Reply:
yes, mainly because there's a few places in the helm chart, but also I think because of getTempFilePath use of hpcc-path.
It could prob. be fixed so can be changed, but perhaps after this.

Reply:
added HPCC-25526